src/main/java/chess/Board.java

```diff
@@ -0,0 +1,63 @@
+package chess;
+
+import piece.Pawn;
+import piece.Piece;
+import piece.Pieces;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Board {
+
+    private static List<List<Piece>> BOARD = intialize();
+    public static final int COUNT_OF_PIECES = 8;
+    public static final int ROW_OF_BLACKPAWN = 1;
+    public static final int ROW_OF_WHITEPAWN = 6;
+    public static final String BLACK = "black";
+    public static final String WHITE = "white";
+
+
+    private static List<List<Piece>> intialize() {
```

Comment:
```suggestion
    private static void intialize() {
```

멤버변수가 있는데 리턴할 필요가 없겠죠?

Reply:
내용 수정했습니다! intialize를 여러번 사용할 필요가 없도록 멤버변수로 캐싱해두고 get하는 방식을 썼었는데, 메서드 가독성이 좋지 않은거 같아서 멤버변수에 직접 add 해준는 방식을 썼습니다. initialize를 외부에서 계속 쓰게 하는 것도 좋은 선택지가 아닌 거 같아서 static 블록으로 처음에 실행하도록 했습니다.